Core/GDCore/Events/Builtin/CommentEvent.cpp

```diff
@@ -12,19 +12,27 @@ using namespace std;
 
 namespace gd {
 
-vector<const gd::EventsList*> CommentEvent::GetAllCommentsVectors()
+vector<const gd::String*> CommentEvent::GetAllSearchableStrings( BaseEvent &event)

```

Comment:
I suppose all this line is wrong. Or at least the end in parenthesis.
I don't know.

Reply:
This method is a method of CommentEvent, so you probably don't need event. Also, return a vector containing strings directly (not pointers to strings).

```suggestion
vector<gd::String> CommentEvent::GetAllSearchableStrings()
```